Gemfile

```diff
@@ -3,6 +3,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.2'
 
+gem 'wdm', '>= 0.1.0'
```

Comment:
When we tryied to make the base project we got some issues at the momment to run the rake db:migrate command, searching on internet we foun that installing this gem the problem it would be solved and yes, now we deleted this gem and ran the app and everything is okay... so, I don't know what was the problem initially🤔

Reply:
I see. Well, if the app works just as well without it, then let's remove it. The fewer the gems, the better! Helps to keep [https://www.productplan.com/glossary/technical-debt/](technical debt) down.